setup.py

```diff
@@ -7,7 +7,7 @@
 version_path = os.path.join(here, "onvif/version.txt")
 version = open(version_path).read().strip()
 
-requires = ["httpx==0.16.1", "zeep[async]==4.0.0"]
+requires = ["httpx==0.18.0", "zeep[async]==4.0.0"]
```

Comment:
Please avoid pinning requirements to exact versions. Pinning to exact versions makes it hard for this library to stay compatible with other libraries installed in the same environment.

This is especially a problem in Home Assistant that has many installed libraries that share the same requirements.

Please set a minimum version and no higher version or a minimum and higher version that is as forgiving as possible.

Reply:
This will work  `httpx>=0.16.1, <0.18.1`

But if you look at libraries.io, this package is used mostly by homeassistant related project. Since latest homeassistant pin httpx to 0.18.0, all homeassistant based project will have to update their httpx to 0.18.0. 

This can be released as a new python package version and if any related project still need the old version, they still can use onvif-zeep-async=1.0.0

Reply:
Have a look at the latest commit